.github/workflows/build-release.yml

```diff
@@ -37,11 +37,13 @@ jobs:
         include:
           - {name: '3.8', python: '3.8', tox: py38}
           - {name: '3.11', python: '3.11', tox: py311}
+          - {name: '3.12', python: '3.12', tox: py312}
```

Comment:
@lafrech Should we drop 3.11 out of the test matrix to keep it min/max?

Reply:
I always do. This time it is a bit different, it is a pre-release but this has no consequence, not like if the pre-release success status was optional, for instance.

Yes, let's drop Python 3.11 here and we'll have no change to make when 3.12 is finally out.